Gruntfile.js

```diff
@@ -230,30 +223,16 @@ module.exports = (grunt) => {
   grunt.registerTask('browserify-components', '',  function() {
     Bluebird.each(fs.readdirSync('components'), (component) => {
       return new Bluebird((resolve, reject) => {
-        const b = browserify({
-          bundleExternal: false,
-          debug: true
-        });
         const src = `./components/${component}/${component}.js`;
         if (!fs.existsSync(src)) {
-          grunt.log.error(`${src} does not exist. If this component is obsolete, please remove that directory or perform a clean build.`);
+          grunt.log.warn(`${src} does not exist. If this component is obsolete, please remove that directory or perform a clean build.`);
+          resolve();
           return;
         }
-        b.add(src);
-        b.external(['ungit-components',
-                'ungit-program-events',
-                'ungit-navigation',
-                'ungit-storage',
-                'ungit-main',
-                'ungit-address-parser',
-                'knockout',
-                'lodash',
-                'hasher',
-                'crossroads',
-                'async',
-                'moment',
-                'blueimp-md5']);
```

Comment:
With `bundleExternal: false`, all external dependencies are skipped, so we don't need this list.